x-pack/test/api_integration/apis/synthetics/add_edit_params.ts

```diff
@@ -4,13 +4,19 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+
 import { v4 as uuidv4 } from 'uuid';
+import { pick } from 'lodash';
 import { SYNTHETICS_API_URLS } from '@kbn/synthetics-plugin/common/constants';
 import expect from '@kbn/expect';
 import { syntheticsParamType } from '@kbn/synthetics-plugin/common/types/saved_objects';
 import { FtrProviderContext } from '../../ftr_provider_context';
 import { PrivateLocationTestService } from './services/private_location_test_service';
 
+function assertHas(actual: unknown, expected: object) {
```

Comment:
I couldn't find a good out-of-the-box assertion that did this. I want to say `expect(actual).to.contain.all.keys.and.values.of(expected)`. If there's a cleaner way that lets me remove this helper please LMK.